scripts/tests/secrets-integration.sh

```diff
@@ -79,8 +79,12 @@ EOF
 "$earthly" secrets --org manitou-org --project earthly-core-integration-test set my_test_file "secret-value"
 "$earthly" secrets --org manitou-org --project earthly-core-integration-test get my_test_file | acbgrep 'secret-value'
 
-# test secrets with org selected in config file
+# test selecting org
 "$earthly" org select manitou-org
+"$earthly" org ls | acbgrep '^\* \+manitou-org'
```

Comment:
does this work? or do you need to disable color?

Reply:
It seemed to work out of the box for this one. I think we are doing something fancy with colors in the `satellite ls` output to be able to gray-out those which come from CI runners.

Reply:
most bizarre! I thought you ran into other issues that were related to grep and color..... but if it works, it works.